### ISSUE TITLE
Drive evaluation result

### ISSUE THREAD

**Issue by loopdigga96** [jlcsilva/EfficientUNetPlusPlus#1]
Hi @jlcsilva thanks for your repository and paper. It was really interesting to read it but I cannot find any evaluation of your approach on public datasets. You mentioned DRIVE in repository but what is actual performance on this dataset?

**Comment by jlcsilva** [comment]
Hey @loopdigga96, you're welcome! Unfortunately, I still haven't obtained approval for sharing the coronary artery dataset. Regarding DRIVE, I still haven't had time to adjust parameters and train it. Due to the lower amount of data (~20 images vs ~200 images), the augmentation policy and loss function hyperparameters will probably need to be fine-tuned for optimal performance. I'm currently finishing a some other work, but I hope to be able to train it soon. I will get back to you with results when I do! 

However, taking into account the 0.8149 F1-score of the ResUNet, which the EfficientUNet++ significantly outperforms in the coronary artery segmentation task, I would expect the EfficientUNet++ to perform better or at least as well as the ResUNet (while being more computationally efficient!)

**Comment by loopdigga96** [comment]
Thanks for you response @jlcsilva! It would be great to hear the results from you.
